packages/plugin-aws-elasticsearch/README.md

```diff
@@ -0,0 +1,24 @@
+# Microfleet AWS-Elasticsearch Plugin
+
+Adds AWS Elastic service support to microfleet. 
+
+## Install
+
+`yarn add @microfleet/plugin-aws-elasticsearch`
+
+## Configuration
+
+To make use of the plugin, adjust microfleet configuration in the following way:
+
+```ts
+exports.plugins = [
+  ...,
+  'awsElasticsearch': {
+    'awsAccessKey',
+    'awsSecretKey',
+    'node',
+    'region',
+  },
+  ...
+]
+```
```

Comment:
update docs, plugins is just a `string[]` of plugin names to include

configuration must be supplied in

```ts
exports.awsElasticsearch = {
   ... // config
}
```

Also I'm not 100% sure, but we need to check that 'aws-elasticsearch' will be camelcased